packages/browser-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bucketco/browser-sdk",
-  "version": "2.4.1",
+  "version": "2.4.2",
```

Comment:
browser-sdk doesn't use the flag-evaluation package, but i think it's fine to do a new patch release since we moved a bunch of stuff around in here.

Reply:
That was my thinking as well.